tests/nemo_text_processing/zh/test_fraction.py

```diff
@@ -30,11 +30,11 @@ def test_norm_fraction(self, test_input, expected):
         preds = self.normalizer_zh.normalize(test_input)
         assert expected == preds
 
-    inverse_normalizer = InverseNormalizer(lang='zh', cache_dir=CACHE_DIR, overwrite_cache=False)
+    # inverse_normalizer = InverseNormalizer(lang='zh', cache_dir=CACHE_DIR, overwrite_cache=False)
 
-    @parameterized.expand(parse_test_case_file('zh/data_inverse_text_normalization/test_cases_fraction.txt'))
-    @pytest.mark.run_only_on('CPU')
-    @pytest.mark.unit
-    def test_denorm(self, test_input, expected):
-        pred = self.inverse_normalizer.inverse_normalize(test_input, verbose=False)
-        assert pred == expected
+    ##@parameterized.expand(parse_test_case_file('zh/data_inverse_text_normalization/test_cases_fraction.txt'))
+    # @pytest.mark.run_only_on('CPU')

```

Comment:
please remove all unnessary # to make code clean

Reply:
This part is responsible for running ITN test. I had separate PRs for TN and ITN, so I commented the ITN part out temporally. This part is needed.

Reply:
Added